client/src/lib/paymentCalc.ts

```diff
@@ -10,10 +10,41 @@ import { readContractValueBigInt } from "./viem";
 export async function calculatePayment(
   chainId: string,
   publicClient: PublicClient,
-  options: AxiomV2ClientOptions,
```

Comment:
add comment `feeData` is the output of `calculateFeeDataExtended` and expects the `overrideAxiomQueryFee` to already include `overrideAxiomQueryFeeL2` considerations.